src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: #000;
+}
+
+.section {
+  margin: 20px 20px 40px;
+}
+
+.box {
+  border: 1px solid #e95d49;
+}
+
+.blue_box {
+  background-color: #308dc0;
+  padding: 10px;
+  width: 350px;
+}
+
+.yellow_box {
+  background-color: #ffa500;
+  padding: 10px;
+}
+
+.white_box {
+  background-color: #ddd;
+  padding: 10px;
```

Comment:
you can just add padding: 10px to the box element

Reply:
Indeed! Thanks